src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goRight(number = 1) {
+      if (number < 0) {
+        return this;
+      }
+      this.coords.x += number;
+
+      return this;
```

Comment:
```suggestion
      if (number > 0) {
        this.coords.x += number;
      }
      
      return this;
```
You can rewrite it in every "go" method.

Reply:
Thank you!